Tests/UnitTests/Core/TrackFitting/KalmanFitterTests.cpp

```diff
@@ -80,6 +80,29 @@ struct TestOutlierFinder {
   }
 };
 
+/// Determine if the smoothing of a track should be done with or without reverse
+/// filtering
+struct TestReverseFilteringLogic {
+  double momentumMax = std::numeric_limits<double>::max();
+
+  /// Classify a measurement as a valid one or an outlier.
+  ///
+  /// @tparam track_state_t Type of the track state
+  /// @param state The track state to classify
+  /// @retval False if the measurement is not an outlier
+  /// @retval True if the measurement is an outlier
+  template <typename track_state_t>
+  bool operator()(const track_state_t& trackState,
```

Comment:
Same comments as for the`VoidReverseFilteringLogic`

Reply:
Done